.gitignore

```diff
@@ -264,3 +264,5 @@ BenchmarkDotNet.Artifacts/
 BDN.Generated/
 binaries/
 
+# Do not exclude any external files
+!external/**/*
```

Comment:
Is this required to override the gitignore of .dll/etc?
https://github.com/dotnet/corefxlab/blob/master/.gitignore#L10
https://github.com/dotnet/corefxlab/blob/master/.gitignore#L51

Do we want to keep this exclusion outside of this single change?